plugins/pyembed/pyembed.cpp

```diff
@@ -1178,8 +1189,8 @@ class GILBlock
 class PythonRowStream : public CInterfaceOf<IRowStream>
 {
 public:
-    PythonRowStream(PythonThreadContext *_sharedCtx, PyObject *result, IEngineRowAllocator *_resultAllocator)
-    : sharedCtx(_sharedCtx), resultIterator(NULL)
+    PythonRowStream(PyObject *result, IEngineRowAllocator *_resultAllocator)
+    : resultIterator(NULL)
```

Comment:
(the prior code code was safe at create time, but not at use time (could be on a different thread) and definitely not at stop() time (since it never got the lock at all))